packages/mui-material-nextjs/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/material-nextjs",
-  "version": "5.15.11",
+  "version": "5.16.1",
```

Comment:
This is not the expected version but I used the stable public version. I think it makes sense if `@mui/material-nextjs` is stable, but let me know if I'm wrong.

Reply:
Agreed